meson.build

```diff
@@ -0,0 +1,27 @@
+project('tinyendian', 'd')
```

Comment:
`subproject_dir` isn't needed, as there are no subprojects and the wrap-system isn't used (see http://mesonbuild.com/Subprojects.html )
But defining a Meson version wouldn't actually be a bad idea - some features used in this file are only available since Meson 0.36, and it only works relaibly with DMD since 0.40.